content/docs/pulumi-cloud/get-started/what-is-it.md

```diff
@@ -0,0 +1,111 @@
+
+{{% notes "info" %}}
+An analogy to other software you may be familiar with is that, just like Git is fully open source and you can host, secure, and manage repositories yourself, so too can you with hosting your Pulumi infrastructure as code projects and stacks. It is much easier, especially in a team, however, to use Git in conjunction with GitHub, as it features easy security, reliability, and collaboration, as well as powerful features like Pull Requests. A similar dynamic exists with Pulumi and Pulumi Cloud.
+{{% /notes %}}
+
+Pulumi Cloud is a platform of three products: [Pulumi IaC](/product/infrastructure-as-code), the companion to Pulumi open source IaC; [Pulumi ESC](/product/secrets-management), a secrets and configuration solution; and [Pulumi Insights](/product/pulumi-insights), an intelligent cloud inventory, compliance, and management product. Pulumi Cloud is offered as a service (SaaS) as well as a self-hosted edition that you can run anywhere. It features a novel client/server architecture to increase your confidence in using the SaaS.
+
+## Five Key Areas
+
+There are five key areas where Pulumi Cloud adds value beyond what is available in the Pulumi IaC open source software:
+
+* **Instant Collaboration**: provides a common place where developers, security experts, and infrastructure practitioners meet to automate, secure, and manage cloud infrastructure
+* **Automatic Security**: ensures security best practices are built-in from the outset
+* **Robustness, Performance, and Scalability** - automatically ensures reliability and scalability as your cloud needs grow, minimizing outages and the need to rearchitect your system over time
+* **Governance and Extensibility**: guarantees that teammates are following your established practices early and always, with full visibility
+* **Cost Effectiveness**: the lowest total cost to maximize return on your cloud investments
+
+Let's examine each of these areas in more detail.
+
+### 1/ Instant Collaboration
+
+The first thing you will notice with Pulumi Cloud is that all of your [projects, stacks, and resources](/docs/pulumi-cloud/projects-and-stacks/) are easy to see, search, and explore. A complete history of who has changed what, when, and how -- with full resource change diffs and links both into the source changes that triggered a deployment as well as forward links to the resources in your cloud consoles -- is always present.
+
+All actions taken by teammates on Pulumi Cloud are logged for [full auditability](/docs/pulumi-cloud/admin/audit-logs/). Full deployment logs are also captured and easy to review to facilitate debugging failures. This is particularly useful for unattended deployments as is common with the [Pulumi Automation API](/automation) and [Pulumi Kubernetes Operator](/docs/iac/using-pulumi/continuous-delivery/pulumi-kubernetes-operator/). All deployment history for all time is maintained and organized.
```

Comment:
unattended sounds like something went rogue

maybe "non-human deployments" sorta like how there is "non-human identities (NIH)" in the security space, meaning aka machines or agents